### ISSUE TITLE
include level sudo_user doesn't work for files

### ISSUE THREAD

**Issue by graingert** [ansible/ansible#10572]
If I  use `inlcude` with `sudo_user` set:

``` yml
# main.yml
- include: doit.yml
  sudo_user: testuser
```

``` yml
# doit.yml
- name: user
  sudo_user: root
  user: name=testuser

- name: implicit sudo user directory
  file: path=/home/testuser/implicit_sudo_user
        state=directory

- name: explicit sudo user directory
  file: path=/home/testuser/explicit_sudo_user
        state=directory
  sudo_user: testuser
```

/home/testuser/implicit_sudo_user is owned by root and /home/testuser/explicit_sudo_user is owned by 'testuser'

I've created a demo of this at https://github.com/graingert/include-sudo_user

``` bash
git clone git@github.com:graingert/include-sudo_user.git
cd include-sudo_user
vagrant up
vagrant ssh
sudo su testuser
cd /home/testuser
ls -altr
$ ls -altr
total 36
-rw-r--r-- 1 testuser testuser  675 Oct  7 15:15 .profile
-rw-r--r-- 1 testuser testuser 3760 Oct  7 15:15 .bashrc
-rw-r--r-- 1 testuser testuser  220 Oct  7 15:15 .bash_logout
drwxr-xr-x 5 root     root     4096 Mar 30 14:43 ..
drwxr-xr-x 2 root     root     4096 Mar 30 14:43 .ssh
drwxr-xr-x 2 root     root     4096 Mar 30 14:46 implicit_sudo_user
drwxr-xr-x 2 testuser testuser 4096 Mar 30 14:46 explicit_sudo_user
drwxr-xr-x 5 testuser testuser 4096 Mar 30 14:46 .
-rw------- 1 testuser testuser   65 Mar 30 14:46 .bash_history
```

**Comment by sivel** [comment]
I believe this may be resolved by: https://github.com/ansible/ansible/issues/10550 https://github.com/ansible/ansible/pull/10566

**Comment by bcoca** [comment]
dupe of #10550

should be fixed by #10566
